src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.General.ThirdPartyModule": "Importation/Exportation de modules tiers",
+  "FSC.Configuration.General.Permissions.Note": "Ici, les autorisations pour diverses interactions avec Simple Calendar peuvent être ajustées en fonction des rôles de Foundry. Les maîtres de jeu auront toujours ces autorisations.",
+  "FSC.Configuration.General.CalendarPlayerVisible": "Afficher le calendrier",
+  "FSC.Configuration.General.CalendarPlayerVisibleHelp": "Les rôles cochés pourront ouvrir Simple Calendar.",
+  "FSC.Configuration.General.PlayersChangeDateTime": "Modifier la date et l'heure",
+  "FSC.Configuration.General.PlayersChangeDateTimeHelp": "Les rôles cochés pourront changer la date et l'heure de Simple Calendar",
+  "FSC.Configuration.Notes.Title": "Paramètres des notes",
+  "FSC.Configuration.Notes.NoteCategories": "Catégories de notes",
+  "FSC.Configuration.NoteCategories.Name": "Nom de la catégorie de note",
+  "FSC.Configuration.NoteCategories.NameHelp": "Le nom d'affichage de cette catégorie de note.",
+  "FSC.Configuration.NoteCategories.Color": "Note Catégorie Couleur",
+  "FSC.Configuration.NoteCategories.ColorHelp": "La couleur associée à la catégorie de note.",
+  "FSC.Configuration.NoteCategories.Add": "Ajouter une catégorie de note",
+  "FSC.Configuration.NoteCategories.RemoveAll": "Supprimer toutes les catégories de notes",
+  "FSC.Configuration.Notes.NoteCategoriesHelp": "Sélectionnez toutes les différentes catégories de notes associées à cette note.",
+  "FSC.Configuration.Year.Title": "Paramètres de l'année",
+  "FSC.Configuration.Year.Current": "Année Courante",
+  "FSC.Configuration.Year.CurrentHelp": "L'année en cours pendant laquelle le jeu se déroule. Cela peut également être modifié avec les contrôles de date.",
+  "FSC.Configuration.Year.Prefix": "Année Préfixe",
+  "FSC.Configuration.Year.PrefixHelp": "Texte qui apparaît avant le numéro de l'année.",
+  "FSC.Configuration.Year.Postfix": "Année Suffix",
+  "FSC.Configuration.Year.PostfixHelp": "Texte qui apparaît après le numéro de l'année.",
+  "FSC.Configuration.Year.YearZero": "Année zéro",
+  "FSC.Configuration.Year.YearZeroHelp": "C'est l'année qui est considérée comme l'année zéro pour calculer combien de temps s'est écoulé. Cela n'a généralement pas besoin d'être modifié.",
+  "FSC.Configuration.Year.YearNames": "Noms des années",
+  "FSC.Configuration.Year.YearName": "Nom de l'année",
+  "FSC.Configuration.Year.YearNameAdd": "Ajouter le nom du nouvel an",
+  "FSC.Configuration.Year.YearNameRemove": "Supprimer tous les noms d'année",
+  "FSC.Configuration.Year.YearNamesStart": "Année de début pour les noms",
+  "FSC.Configuration.Year.YearNamesStartHelp": "L'année de début du premier nom de l'année sur la liste.",
+  "FSC.Configuration.Year.YearNameBehaviour": "Comportement des noms d'années",
+  "FSC.Configuration.Year.YearNameBehaviourHelp": "Comment déterminer l'ordre des noms d'année.",
```

Comment:
This translation sounds too literal, but I am also having trouble understanding the context. You can name years? Could you help me with this one @vigoren ?

Reply:
Yes years can have names, the primary Calendar for this is the dark Sun calendar where every year has a name as well as a numeric representation. So these settings are around creating the list of year names then deciding how those names are assigned to a given year and which year the list starts at. 

In order will start at the starting year and go down the list of names given each subsequent year the next name in the list.
Repeats is the same as in order except when the list of names runs out it starts from the beginning of the list again 

Random just randomly assigned a name from the list to a given year

Reply:
Ok! Thanks for the clarification.

This part does not require any changes